src/models/serialization.jl

```diff
@@ -114,6 +114,15 @@ function deserialize_uuid_handling(field_type, val, component_cache)
             value = _vals
         else
             uuid = deserialize(Base.UUID, val)
+            if !haskey(component_cache, uuid)
```

Comment:
This error relates to component composition, not time series. A user would get this error if the bus attached to a generator was removed apart from system methods (unless there was a code bug).

Reply:
This is the location of the error in https://github.com/NREL-Sienna/PowerSystems.jl/issues/1409. Am I getting something wrong?

Reply:
nvm I see now, this is not about time series. This was an Arc